spec/components/collections/workflow_review_component_spec.rb

```diff
@@ -0,0 +1,19 @@
+# typed: false
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Collections::WorkflowReviewComponent, type: :component do
+  let(:rendered) { render_inline(described_class.new(collection: collection)) }
+
+  context 'when displaying a collection' do
+    let(:review_enabled) { 'On' }
```

Comment:
I'm not sure why this is in a let block.  It's only used the one time and I think having to reference up to the let blocks makes the test hard to read.